lesson08/main.js

```diff
@@ -31,18 +31,23 @@ function renderMenus(menus){
   ul.appendChild(fragment);
 }
 
-const getMenus = new Promise((resolve)=> {
+const getMenus = new Promise((resolve, reject)=> {
   renderCircle();
   const menus = [ 
     {to: "bookmark.html", img:"img/1.png", alt:"画像1", text: "ブックマーク"}, 
     {to: "message.html", img:"img/2.png", alt:"画像2", text: "メッセージ"} 
   ]; 
   setTimeout(() => {
-    resolve(menus);
+    reject("error");
   }, 3000);
 })
 
-getMenus.then((menus)=> {
-  removeCircle();
-  renderMenus(menus);
-});
+getMenus.then(
+	(menus)=> {
+  	removeCircle();
+  	renderMenus(menus);
+	},
+	(error) => {
+		console.error("error", error);

```

Comment:
IMO
I think `error` already contains the string "error". 
```suggestion
		console.error(error);

```

Reply:
thank you mai-san :)
I fixed it.
d81c54e9af16596f0f954d20df2947b443747643